cpp/src/io/parquet/page_enc.cu

```diff
@@ -1286,12 +1321,14 @@ static __device__ void byte_reverse128(__int128_t v, void* dst)
                d_char_ptr);
 }
 
-__device__ uint8_t* EncodeStatistics(uint8_t* start,
-                                     const statistics_chunk* s,
-                                     uint8_t dtype,
-                                     void* scratch)
+__device__ void get_extremum(const statistics_chunk* s,
+                             uint8_t dtype,
+                             bool is_min,
+                             void* scratch,
+                             const void** val,
+                             uint32_t* len)
```

Comment:
Oh, duh.  That's way better.

Reply:
Change made, but now I remember why is_min was there in the first place.  It's needed for stats truncation because min and max truncate differently.  Still like passing the statistics_val so that stays!